regression/test.pl

```diff
@@ -304,10 +304,15 @@ ($$$$)
 $main::VERSION = 0.1;
 $Getopt::Std::STANDARD_HELP_VERSION = 1;
 our ($opt_c, $opt_i, $opt_j, $opt_n, $opt_p, $opt_h, $opt_C, $opt_T, $opt_F, $opt_K, %defines, @include_tags, @exclude_tags); # the variables for getopt
-$opt_j = 0;
 GetOptions("D=s" => \%defines, "X=s" => \@exclude_tags, "I=s" => \@include_tags);
 getopts('c:i:j:nphCTFK') or &main::HELP_MESSAGE(\*STDOUT, "", $main::VERSION, "");
 $opt_c or &main::HELP_MESSAGE(\*STDOUT, "", $main::VERSION, "");
+$opt_j = $opt_j || $ENV{'TESTPL_JOBS'} || 0;
+if($opt_j && !$has_thread_pool) {
```

Comment:
Test for `$opt_j > 1` as `-j 1` is fine even without a thread pool :-)

Reply:
I'm not opposed to this in general, but note that the next line causes the help message to print and exit if -j is used without Thread::Pool::Simple, so if I'm changing this I should change that too.

Reply:
Good point. I'd suggest: change `print` to `warning` in your newly added message, change that to `> 1` as I suggested, and completely remove the exit below. It's perfectly safe to proceed even when `Thread::Pool::Simple` is not available, the user just wouldn't get any parallelism.